Test bootstrap

Have-Samuel opened the pull request:
**Test bootstrap**

Testing bootstrap concepts

Have-Samuel committed:
Added a background-image and modified HTML & CSS files

Have-Samuel committed:
Aligned and center the fast element of the section

Sirvelasque commented on index.html:
```diff
@@ -0,0 +1,72 @@
+  </header>
+  <section class="first-layout">
+    <div class="container col-sm-5 mt-5">
+      <div class="hey-there">
+        <h1>Hey There. I'm Have <br> <span>I'm a software developer</span></h1>
+      </div>
```
[OPTIONAL]
I would recommend to add a class to the span item  in order to fit better to the template

Sirvelasque reviewed (commented):
Nice @Have-Samuel 
Good job so far!
There are some issues that you should work on
# Required changes ♻️ 
Check the comments below and under the review.
# Optional sugestion
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Have-Samuel committed:
Added a scrollspy to the page

Have-Samuel committed:
Added images & modified HTML & CSS

Have-Samuel committed:
Modified HTML & CSS and added some images

Have-Samuel committed:
Modified HTML & CSS file

Sirvelasque committed:
set font roboto

Sirvelasque committed:
start about section

make support text and "get my resume button

Sirvelasque committed:
made the first about sub-container

made the container for languages and properly set up some classes

Sirvelasque committed:
make a copy and edit them

made a copy of the languages box and edit them

Sirvelasque committed:
put box badges as h4 and raunded  border

put as a title to increase the size and rounder-pill shape for the budgets surrounds

Sirvelasque committed:
add margin, change font size and width for section elements

Sirvelasque committed:
add images folder and form section

add contact us section main structure

Sirvelasque committed:
sett form classes, properties and section botton margin

changes text area width o make it responsive

Sirvelasque committed:
apply breakpoint

Have-Samuel committed:
Edited the index.html file
